drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -19,6 +19,12 @@ def ssn_serial
 
   SORT_OPTIONS = [:last_name_asc, :last_name_desc].freeze
 
+  def self.client_search(input:, _user: nil)
+    scope = GrdaWarehouse::Hud::Client.all
```

Comment:
This needs to be .source.where(data_source_id: hmis_data_source_id) instead of all.

Reply:
this should also have a `searchable_by(user)` on `GrdaWarehouse::Hmis::Client` which we can stub in for now.